Library/Formula/mysql-sandbox.rb

```diff
@@ -11,7 +12,7 @@ class MysqlSandbox < Formula
   end
 
   def install
-    ENV.prepend_create_path "PERL5LIB", "#{HOMEBREW_PREFIX}/lib/perl5/site_perl"
+    ENV.prepend_create_path "PERL5LIB", libexec/"lib/perl5/site_perl/"
 
     system "perl", "Makefile.PL", "PREFIX=#{prefix}"
```

Comment:
If we're going with `libexec` this should be changed to `libexec` as well so everything is still in the same directory, and then [this logic](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/dpkg.rb#L41-L43) used to put the executables back in the top-level directory which'll be symlinked into the `$PATH`.